src/module/vue/components/asset/asset.vue

```diff
@@ -93,6 +93,7 @@ import AssetExclusiveoption from './asset-exclusiveoption.vue'
 import Clock from '../clock.vue'
 import WithRolllisteners from '../with-rolllisteners.vue'
 import { $ActorKey } from '../../provisions'
+import { defaultActor } from '../../../helpers/actors'
```

Comment:
hmm. i've run into a few long relative paths like this in my meter branch, too. reckon it'd make sense to introduce some absolute paths?

Reply:
Yeah, maybe that `@/` alias would be good here. Not sure if VS Code will route automatic imports through that though.